Packs/CommonScripts/ReleaseNotes/1_10_30.md

```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### New: CreateNewIndicatorsOnly
+- Create new indicators to the Threat Intel DB only if they are not registered. (Available from Cortex XSOAR 6.5.0).
```

Comment:
- Added a  script (createNewIndicatorsOnly) to allow a user to only create indicators that are not already present in the DB (Available from Cortex XSOAR 6.5.0).